bundles/org.openhab.core.automation.module.script/src/test/java/org/openhab/core/automation/module/script/ScriptTransformationServiceTest.java

```diff
@@ -65,56 +70,64 @@ public class ScriptTransformationServiceTest {
     public void setUp() throws ScriptException {
         service = new ScriptTransformationService(transformationConfigurationRegistry, scriptEngineManager);
 
-        when(scriptEngineManager.createScriptEngine(eq(SCRIPT_TYPE), any())).thenReturn(scriptEngineContainer);
+        when(scriptEngineManager.createScriptEngine(eq(SCRIPT_LANGUAGE), any())).thenReturn(scriptEngineContainer);
         when(scriptEngineManager.isSupported(anyString()))
-                .thenAnswer(scriptType -> SCRIPT_TYPE.equals(scriptType.getArgument(0)));
+                .thenAnswer(scriptType -> SCRIPT_LANGUAGE.equals(scriptType.getArgument(0)));
```

Comment:
Script language is here compared to script type now - could the naming be aligned?